app/helpers/decidim/vocdoni/admin/steps_wizard_helper.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Vocdoni
+    module Admin
+      # Custom helpers, scoped to the vocdoni admin engine.
+      #
+      module StepsWizardHelper
+        def tab_class(tab_name, active_class)
+          css_classes = ["tabs-title"]
+          css_classes << "is-active" if active_class == tab_name
+          css_classes.join(" ")
+        end
+
+        # rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
+        def tab_link_class(tab_name, election)
+          case tab_name
+          when "questions"
+            election.present? ? "" : "disabled"
+          when "census"
+            election&.minimum_answers? || census_status == "ready" ? "" : "disabled"
+          when "results"
+            census_status == "ready" ? "" : "disabled"
+          when "publish"
+            election.start_time.present? && election.end_time.present? && census_status == "ready" ? "" : "disabled"
```

Comment:
We can probably remove the `Metrics/CyclomaticComplexity` cop exception if we refactor this by extracting some of the logic to private methods, for instance: 

```suggestion
            is_publishable? ? "" : "disabled"
```